perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java

```diff
@@ -391,6 +392,45 @@ public void deleteUserWhenUserNotExists() throws Exception {
 		// shouldn't find user
 	}
 
+	@Test
+	public void anonymizeUser() throws Exception {
```

Comment:
If the default set of anonymized attributes would be changed (anonymized only uids and logins by default) you need to also change the test accordingly.

Reply:
The attributesToAnonymize property is changed in the test to contain only mail attribute. After anonymizing the user in the test, the property is set back to the original attribute set.